generator/internal/discovery_resource.h

```diff
@@ -56,11 +64,14 @@ class DiscoveryResource {
   // are concatenated with the resource name.
   std::string FormatMethodName(std::string method_name) const;
 
+  StatusOr<std::string> JsonToProtoService() const;
```

Comment:
given that there is no json argument to this function, should we call it: `ToProtoService()` ?